### ISSUE TITLE
Using sigmoid instead of softmax

### ISSUE THREAD

**Issue by inspirit** [brantbzhang/tensorflow-mtcnn#1]
Hello, i was wondering what is the reason for using sigmoid activation instead of softmax:
https://github.com/brantbzhang/tensorflow-mtcnn/blob/master/core/model.py#L47

**Comment by brantbzhang** [comment]
sigmoid activation output shape (batch_size,)
softmax  output shape (batch_size,2)
I just think use sigmoid is easy to compute hard mining loss.
